config.yml

```
@@ -0,0 +1,50 @@
+---
+variables:
+  PRODUCT: &PRODUCT 90427
+  BASE_URL: &BASE_URL /path/to/product/build/doc/    # CHANGE THIS
+
+project: MLX{product}
+
+json_folder: [*BASE_URL, *PRODUCT, /database.json]
+# URL: [*BASE_URL, *PRODUCT, /rom/html/document.html#id]
+URL: [*BASE_URL, *PRODUCT, /html/document.html#id]
+
+# Keep this order !
+filters:
+  - SYSRQT
+  - SWRQT
+  - SWARCH
+  - DESIGN
+  - QTEST
+  - SWITEST
+  - UTEST
+
+# Keep this order !
+group_colours:
+  SYSRQT: "#fc851c"
+  SWRQT: "#FF33CC"
+  SWARCH: "#7777FF"
+  DESIGN: "#cf5402"
+  QTEST: "#00FF00"
+  SWITEST: "#00FF00"
+  UTEST: "#00FF00"
+
+link_colours:
```

```suggestion
link_colors:
```
Let's use American English. Please update all other occurrences. Or does a  dependency like D3js use "colour" everywhere?

---

Resolved in 83a206dcb8dc459fb807e351a7600c2d92024b89